.travis.yml

```diff
@@ -9,9 +9,9 @@ env:
 
 python:
   - 2.7
-  - 3.3
   - 3.4
   - 3.5
+  - 3.6
```

Comment:
👍 Definitely time to drop 3.3. Nice to see 3.6.

Reply:
At some point, we should add Python 3.7, but that is currently blocked on issue ( https://github.com/travis-ci/travis-ci/issues/9815 ).